src/nunavut/lang/c/templates/_composite_type.j2

```diff
@@ -88,6 +88,6 @@ inline void {{ composite_type | full_reference_name }}_init({{ composite_type |
  * Contains the maximum number of bytes needed to serialize a(n) {{ composite_type | full_reference_name }}
```

Comment:
Do we really need the `@def` line in the doc comment? It's redundant, I suggest we remove it.

Reply:
Agreed.